spec/02-integration/05-proxy/11-error_default_type_spec.lua

```diff
@@ -40,179 +40,90 @@ describe("error_default_type", function()
     end
   end)
 
-  describe("request `Accept` is missing", function()
-    describe("(default)", function()
-      setup(function()
-        assert(helpers.kong_exec("reload", {
-          prefix             = helpers.test_conf.prefix,
-          nginx_conf         = "spec/fixtures/custom_nginx.template",
-        }))
-
-        ngx.sleep(RELOAD_DELAY)
-      end)
+  it("no Accept header uses error_default_type", function()
+    local res = assert(client:send {
+      method  = "GET",
+      path    = "/",
+      headers = {
+        accept = nil,
+      }
+    })
 
-      it("returns error messages in plain text", function()
-        local res = assert(client:send {
-          method  = "GET",
-          path    = "/",
-          headers = {
-            accept = nil,
-            host   = "example.com",
-          }
-        })
+    local body = assert.res_status(RESPONSE_CODE, res)
```

Comment:
Would be nice to see an assertion of the `res` content type header as well for completeness sake, but lgtm overall.

Reply:
Agreed; focus was more on tests performance, but will add it once this is confirmed to be more consistent on Travis.